### DISCUSSION TITLE
Do the same Renovate instance can operate on both Bitbucket and GitHub

### DISCUSSION THREAD

**Discussion by brimac2** [renovatebot/renovate#30932] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

bitbucket-server 37.421.3

### Please tell us more about your question or problem

We currently have Renovate running on a Helm chart, integrated with Bitbucket. Our team is migrating from Bitbucket to GitHub, and we plan to use the same Renovate configuration (deployed via Helm chart) for GitHub.

However, we noticed that the platform key in the Renovate configuration appears to accept only a single string value, not an array. Is it possible to configure Renovate to work with multiple platforms simultaneously (e.g., Bitbucket and GitHub) using the same Helm chart deployment? If so, how should we adjust the configuration to accommodate this, or do we need separate deployments for each platform?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
You need to run it separately/once per platform

**Comment by brimac2** [comment]
So the wokflow would be create a separate release/deployment with different config for each platform, right?

**Reply by rarkins** [reply]
You could run it from whatever environment you already do (e.g. pipeline, or a kubernetes environment, or whatever you use) but run it twice - once with the bitbucket config and second with the github config
